content/en/docs/guides/namespace-migration-across-tenants.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Namespace Migration Across Tenants
+weight: 2
+description: "A Step-by-Step Guide to Namespace Migration"
+---
+Capsule relays on two components to associate given namespace with tenant.
+- Namespace's OwnerReference pointing to the Tenant defintion
+- Namespace's label capsule.clastix.io/tenant
+
+If a cluster administrator changes the Namespace by matching the other Tenant with the proper UID and name, along with the labels, the Namespace can be easily transferred.
+
+```bash
+kubectl get tenants
+```
+```
+NAME    STATE    NAMESPACE QUOTA   NAMESPACE COUNT   NODE SELECTOR   AGE
+solar   Active                     1                                 46s
+wind    Active                     1                                 39s
+```
+Get namespace's labels and metadata.uid.
+```bash
+kubectl get ns -l capsule.clastix.io/tenant --show-labels
+kubectl get tnt wind -o jsonpath='{.metadata.uid}'
+```
+```
+NAME     STATUS   AGE   LABELS
+ns-bar   Active   28s   capsule.clastix.io/tenant=wind,kubernetes.io/metadata.name=ns-bar
+ns-foo   Active   36s   capsule.clastix.io/tenant=solar,kubernetes.io/metadata.name=ns-foo
+0df8e9ee-5f6f-40a4-897d-b80d349ca36f%
+```
+Edit UID and labels to match the output of the previous commands.
```

Comment:
Hi, sorry i delyed so much to start the document. 
About the procedure, i think we only need to change the tenant name and UID in ownerReferences, the capsule tenant label will sync once we change the ownerReferences.  
And also, i don't know it's intended or not, you can change the ownership by only change the tenant name in ownerReferences, which will leave the tenant name and tenant uid conflict, but capsule works fine. Seems capsule respect the tenant name rather than uid.

Reply:
> bout the procedure, i think we only need to change the tenant name and UID in ownerReferences, the capsule tenant label will be changed automatically once we change the ownerReferences.

Sounds interesting, I will give it a try and I will update the documentation afterwards.

Reply:
@shadow625 I've updated the wording, let me know if it looks good to you.